src/components/todoButtonsContainer/index.js

```diff
@@ -12,7 +12,7 @@ import {
 import { calculateDate } from "utils/calculateDate"
 import "components/todoButtonsContainer/todoButtonsContainer.scss"
```

Comment:
Import section should be sorted like this,
Library functions -> Files -> Components.